packages/lesswrong/components/dropdowns/posts/PostActions.tsx

```diff
@@ -77,10 +77,10 @@ const PostActions = ({post, closeMenu, includeBookmark=true, classes}: {
       />
       <NotifyMeDropdownItem
         document={post}
-        enabled={!!post.debate}
+        enabled={!!post.collabEditorDialogue}
```

Comment:
Note that this means users can no longer subscribe to old dialogues

Reply:
Yeah, that seems fine